src/scripts/preparation/Merge_data.R

```diff
@@ -0,0 +1,278 @@
+########################################################################
+## Script name: Merge_data
+## Purpose of script: Creatw a name-matching table to merge the statistic 
+## data downloaded from INEI website with district shapefile in Peru
+## Author: Chenyu Shen
+## Date Created: 2024-1-27
+########################################################################
+
+### =========================================================================
+### A - Preparation
+### =========================================================================
+## Install and load packages
+packs <- c("sf", "dplyr", "stringi")
+
+# Install new packages
+new.packs <- packs[!(packs %in% installed.packages()[, "Package"])]
+if (length(new.packs)) install.packages(new.packs)
+
+# Load required packages
+invisible(lapply(packs, require, character.only = TRUE))
+
+# Set working directory
+setwd(Data_dir)
```

Comment:
As we move code the github we need to be careful with hard-coding dirs and the working_dir. Better to rely on the files being in the same location as the .Rproj file on your local computer so then the Rproj. is the working directory and any files/dir are relative to this.